lib/handlers/router-entities/route-caching/dynamo-route-caching-provider.ts

```diff
@@ -224,8 +225,8 @@ export class DynamoRouteCachingProvider extends IRouteCachingProvider {
       const putParams = {
         TableName: this.tableName,
         Item: {
-          pairTradeTypeChainId: partitionKey.toString(),
-          protocolsBucketBlockNumber: sortKey.fullKey(),
+          [DynamoDBTableProps.CacheRouteDynamoDbTable.PartitionKeyName]: partitionKey.toString(),
+          [DynamoDBTableProps.CacheRouteDynamoDbTable.SortKeyName]: sortKey.fullKey(),
           item: binaryCachedRoutes,
           ttl: ttl,
```

Comment:
Should we do the same here with the ttl attribute?

Reply:
yep